repos/tap/utils/getConfig.js

```diff
@@ -1,3 +1,3 @@
-import * as Config from '../../../configs/app.config.js'
+import * as Config from '../../../configs/herkin.default.config.js'
```

Comment:
We need a different way to do this. The `herkin.default.config.js` file `requires` the `path` library which is a native `node` package.
* Most likely we're not seeing any issues with this, because webpack 4 auto polyfills and native node packages. 
* This has been changed in webpack 5, so in that case we would see an error about it.

Anyway, the only reason the tap needs this is to get the API backend url. We can do this in other ways, like through an ENV, and the webpack `DefinePlugin` plugin, which I know you're familiar with

Reply:
Updated to use the `tap.js` envs plugin